### DISCUSSION TITLE
How to enable FreeU on startup

### DISCUSSION THREAD

**Discussion by neuraloutput** [lllyasviel/Fooocus#2655] [Q&A] [answered]
Config tutorial don't have such line, i can't figure it out myself.

**Comment by ILikeToasters** [comment]
I don't think there is a setting for that. You would probably have to edit the code yourself.

**Comment by IPv6** [comment]
Not possible, but really needed...

Would be cool to have preset options for "Image" checkbox, "Debug features" checkbox, "Mix inpaint/etc with prompt" checkboxes, "Advanced use" checkboxes and "FreeU enabled" checkbox. Ability to set this via preset-config would be really helpful, to avoid tedious actions on each startup

**Reply by Rogala** [reply, marked as answer]
Edit the `webui.py` file to find all the switches you need .

385 string:
dev_mode = gr.Checkbox(label='Developer Debug Mode', value=__True__, container=False)
with gr.Column(visible=__True__) as dev_tools:
....
and then down the line, include whatever you need.
....
513 string:
with gr.Tab(label='FreeU'):
                        freeu_enabled = gr.Checkbox(label='Enabled', value=__True__)

**Reply by IPv6** [reply]
> Edit the `webui.py` file to find all the switches you need

Thanks for details! Still would be good to have this in preset - for colab usage, where py editing is not a convenient option
